package.json

```
@@ -49,6 +49,6 @@
     "rollup-plugin-terser": "^7.0.2"
   },
   "peerDependencies": {
-    "chart.js": "^3.0.0"
+    "chart.js": ">=2.8.0"
```

Why not exclude 4.x?
```suggestion
    "chart.js": ">=2.8.0 < 4"
```

---

Why exclude? I think its a greater change this will work with 4.x than not. That translates directly to needing to do a new release of this adapter when 4.x is released.

---

Just avoid unexpected breaking changes in chart.js 4.x. You can include 4.x in a next patch release 😁

---

Yeah I agree with @kurkle, especially considering there isn't any version 4 released, and I haven't seen any plans to release a new major version as far as I know. I think it would be great to leave it as is to prevent the least amount of disruption 😄